packages/react/src/page-alert/PageAlert.tsx

```diff
@@ -57,9 +60,10 @@ export const PageAlert = forwardRef<HTMLDivElement, PageAlertProps>(
 		},
 		forwardedRef
 	) {
-		const ref = useFocusOnMount<HTMLDivElement>({
-			disabled: !focusOnMount,
+		const ref = useFocus<HTMLDivElement>({
+			disabled: !focusOnMount && !focusOnUpdate,
```

Comment:
Ah, this is how we use `disabled`. It feels less clear now their are 2 conditions and also makes consumers have to do this every time. Why don't we pass `focusOnMount` to the hook and let the hook work it out?